src/blocks/posts/inspector.js

```diff
@@ -111,9 +130,9 @@ const Inspector = props => {
 					value={ columns }
 					onChange={ ( value ) => {
 						onUserModifiedColumn();
-						setAttributes( { columns: value } );
+						columnCountByPosts( value );
 					} }
-					min={ isStackedStyle ? 2 : 1 }
+					min={ isStackedStyle ? Math.min( 2, postCount ) : 1 }
```

Comment:
See my comments in the [`post-carousel/inspector.js`](https://github.com/godaddy/coblocks/pull/1084/files#diff-2e5f26fcb8648e88a57c0ffad902109b) file as they apply here as well.